### ISSUE TITLE
Stepsize changes on a day that includes daylight savings change.

### ISSUE THREAD

**Issue by bachandi** [mikaku/Monitorix#361]
Yesterday, during the daylight savings transition, I noticed that the step size for all daily plots changed to about 30 minutes. The days before and after which did not include the daylight savings change had the normal finder step size.

Day with change from summer to winter time:
![system1 1day_daylight](https://user-images.githubusercontent.com/62039342/139639899-864393cb-bdc3-4b4e-8d54-b9eb15c6a2cb.png)

A normal day that does not have the daylight change in the time range:
![system1 1day](https://user-images.githubusercontent.com/62039342/139639910-803668c4-4b29-40b4-8db6-3f745a4aa746.png)

I am not really sure if it is related to monitorix or a feature in RRDtool (Version 1.7.2). Is this a know issue?

**Comment by mikaku** [comment]
Yes, I've seen this effect since the beginning, only in October.

This is probably because days are defined in `.rrd` files as 24 hours, and during the daylight saving time, in October, such day had actually 25 hours. This can also affect other time frames since all are based on 24-hour days.

**Comment by bachandi** [comment]
Thanks. So there is propably nothing that monitorix can do about it.
